### PR TITLE
Updated Java long audio sample code

### PR DESCRIPTION
**Pull request by cherrycaol**

### CODE REVIEW

**Comment by boltomli** [comment]
@cherrycaol would you replace this with your new PR? If so please close as appropriate. Thanks.

**Comment by cherrycaol** [comment]
> @cherrycaol would you replace this with your new PR? If so please close as appropriate. Thanks.

Yes, thanks!

File: CustomVoice-API-Samples/CSharp/CustomVoice-API/APIArguments.cs [hunk 1]
```diff
@@ -21,7 +21,7 @@ public class APIArguments
         internal const string EndpointId = "endpointId";
         internal const string BatchSynthesisId = "batchSynthesisId";
         internal const string VoiceTestId = "voiceTestId";
-        internal const string HostUriValue = "https://Westus.customvoice.api.speech.microsoft.com/";
+        internal const string HostUriValue = "https://<region>.customvoice.api.speech.microsoft.com/";
```
[the remark this answers is not in the document]
Comment (cherrycaol) [reply CustomVoice-API-Samples/CSharp/CustomVoice-API/APIArguments.cs]: The definition is in lines 459-461 of the current file where your added the first comment

File: CustomVoice-API-Samples/CSharp/CustomVoice-API/ArgumentsDescription.cs [hunk 2]
```diff
@@ -456,6 +456,9 @@ private static void PrintActionUsageBase(string action, string description, stri
             Console.WriteLine("");
             Console.WriteLine($"Sample command : {sampleCommand}");
             Console.WriteLine("");
+            Console.WriteLine("See the link below for a list of supported regions");
+            Console.WriteLine("https://docs.microsoft.com/en-us/azure/cognitive-services/speech-service/regions#speech-to-text-text-to-speech-and-translation");
+            Console.WriteLine("");
             Console.WriteLine("See the link below for a list of ibizastsurl");
             Console.WriteLine("https://docs.microsoft.com/en-us/azure/cognitive-services/speech-service/rest-text-to-speech#how-to-get-an-access-token");
             Console.WriteLine("");
```
[the remark this answers is not in the document]
Comment (cherrycaol) [reply CustomVoice-API-Samples/CSharp/CustomVoice-API/ArgumentsDescription.cs]: I think it's ok, it's more readable.

File: CustomVoice-API-Samples/Java/src/main/java/customvoice/Main.java [hunk 3]
```diff
@@ -0,0 +1,242 @@
...
+			cli = cliParser.parse(options, args);
+		} catch (ParseException e) {
+			helpFormatter.printHelp(">>>>>> parameter options", options);
+			return;
+		}
+
+		String hostPath = cli.getOptionValue("hosturl");
+		String subscriptionKey = cli.getOptionValue("subscriptionkey");
+
+		ApiClient apiClient = new ApiClient(hostPath);
```
[the remark this answers is not in the document]
Comment (cherrycaol) [reply CustomVoice-API-Samples/Java/src/main/java/customvoice/Main.java:40]: This PR is abandoned, fixed this in PR#174

File: CustomVoice-API-Samples/Java/src/main/java/customvoice/VoiceSynthesisApi.java [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+
+	/**
+	 * Gets a list of supported voices for offline synthesis.
+	 * 
+	 * @return List&lt;Voice&gt;
+	 * @throws ApiException If fail to call the API, e.g. server error or cannot
+	 *                      deserialize the response body
+	 */
+	public List<Voice> getSupportedVoicesForVoiceSynthesis() throws ApiException {
+		String localVarPath = Configuration.VoiceSynthesis_BasePath + "/voices";
```
[the remark this answers is not in the document]
Comment (cherrycaol) [reply CustomVoice-API-Samples/Java/src/main/java/customvoice/VoiceSynthesisApi.java:104]: This PR is abandoned, fixed this in [PR#174](https://github.com/Azure-Samples/Cognitive-Speech-TTS/pull/174)

File: CustomVoice-API-Samples/Java/src/main/java/customvoice/VoiceSynthesisApi.java [hunk 5]
```diff
@@ -0,0 +1,180 @@
...
+	 * @param timeStart The timeStart filter
+	 * @param timeEnd   The timeEnd filter
+	 * @param status    The status filter
+	 * @param skip      The skip filter
+	 * @param top       The top filter
+	 * @return List&lt;VoiceSynthesis&gt;
+	 * @throws ApiException If fail to call the API, e.g. server error or cannot
+	 *                      deserialize the response body
+	 */
+	public List<VoiceSynthesis> getVoiceSyntheses(String timeStart, String timeEnd, String status, int skip, int top)
```
[the remark this answers is not in the document]
Comment (cherrycaol) [reply CustomVoice-API-Samples/Java/src/main/java/customvoice/VoiceSynthesisApi.java:123]: This PR is abandoned, fixed this in [PR#174](https://github.com/Azure-Samples/Cognitive-Speech-TTS/pull/174)

File: CustomVoice-API-Samples/Java/src/main/java/customvoice/Main.java [hunk 6]
```diff
@@ -0,0 +1,242 @@
...
+							System.out
+									.println("Please enter a valid top parameter, should be a ingeter greater than 0");
+							return;
+						}
+					} catch (NumberFormatException e) {
+						System.out.println("Please enter a valid top parameter, should be a ingeter greater than 0");
+						return;
+					}
+				}
+				List<VoiceSynthesis> result = api.GetVoiceSynthesis(timeStart, timeEnd, status, skip, top);
```
[the remark this answers is not in the document]
Comment (cherrycaol) [reply CustomVoice-API-Samples/Java/src/main/java/customvoice/Main.java:133]: This PR is abandoned, fixed this in [PR#174](https://github.com/Azure-Samples/Cognitive-Speech-TTS/pull/174)

**Pull request closed without merging**